src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -481,8 +486,8 @@ public void OnTransactionCommitted(LowLevelTransaction tx)
                 action?.Invoke(tx);
             }
 
-            public long LastFlushedTransactionId => Interlocked.Read(ref _lastFlushedTransactionId);
-            public long LastFlushedJournalId => Interlocked.Read(ref _lastFlushedJournalId);
+            public long LastFlushedTransactionId => Interlocked.Read(ref _lastFlushed.TransactionId);
```

Comment:
No, you need:

```suggestion
            public long LastFlushedTransactionId => Interlocked.Read(ref _lastFlushed).TransactionId;
```

In all places.
The values are _never_ mutated

Reply:
So actually I don't need `Interlock` here, right?

Reply:
Yes, that would be fine without it